packages/terra-heading/src/Heading.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import 'terra-base/lib/baseStyles';
+import './Heading.scss';
+
+const propTypes = {
+  /**
+   * Child nodes
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Sets the heading level. One of 1, 2, 3, 4, 5, 6.
+   */
+  level: PropTypes.oneOf([1, 2, 3, 4, 5, 6]).isRequired,
+  /**
+   * The color of the text. Accepts any color value parseable by CSS.
+   */
+  color: PropTypes.string,
+  /**
+   * Sets the text to display in italics.
+   */
+  isItalic: PropTypes.bool,
+  /**
+   * Sets the text to appear visually hidden. The text will still be available to screen readers.
+   */
+  isVisuallyHidden: PropTypes.bool,
+  /**
+   * Sets the text size. One of mini, tiny, small, medium, large, huge.
+   */
+  size: PropTypes.oneOf(['mini', 'tiny', 'small', 'medium', 'large', 'huge']),
+  /**
+   * Sets the text size. One of 200, 400, 700.
+   */
+  weight: PropTypes.oneOf([200, 400, 700]),
+};
+
+const defaultProps = {
+  color: 'inherit',
+  isItalic: false,
+  isVisuallyHidden: false,
+  weight: 700,
+};
+
+const Heading = ({ level, color, children, isVisuallyHidden, isItalic, size, weight, ...customProps }) => {
+  const attributes = Object.assign({}, customProps);
+  const TextClassNames = classNames([
+    'terra-Heading',
+    { 'terra-Heading--italic': isItalic },
+    { 'terra-Heading--visually-hidden': isVisuallyHidden },
+    { [`terra-Heading--level-${level}`]: level },
+    { [`terra-Heading--size-${size}`]: size },
+    { [`terra-Heading--weight-${weight}`]: weight },
+    attributes.className,
+  ]);
+
+  const headingStyles = {
+    color,
```

Comment:
I just remembered these issues:

https://github.com/cerner/terra-core/issues/370
https://github.com/cerner/terra-core/issues/366

Do we still need to allow color to be set with css?